cypress/e2e/tests/administration/jira-connection/crud.test.ts

```diff
@@ -10,66 +14,63 @@ describe(["@tier2"], "CRUD operations for Jira Server connection instance", () =
     const toBeCanceled = true;
     const notToBeCanceled = false;
```

Comment:
Isn't this redundant since we already have `toBeCanceled` ?

Reply:
The reason of it is simple. I wanted to make the code further clear, so when I call constructor of object further - it is written clearly what is the value, like should it be canceled or not.